envsetup.sh

```diff
@@ -12,6 +12,15 @@ if [ ! -e ./opendronemap.iso ]; then
 	./gdown.pl "https://drive.google.com/file/d/$GOOGLE_DRIVE_FILE_ID/edit" opendronemap.iso
 fi
 
+# Clean any previous failed execution.
+set +e
```

Comment:
Can we make this generic? `/home/andres/LiveODM` is a hard-coded path.

Reply:
Done